Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdb/persistence/HibernateUtil.java

```diff
@@ -107,7 +107,9 @@ public static void closeSession(Session session, boolean flush) {
             if (flush) {
                 session.flush();
             }
-            session.close();
+            if (session.isOpen()) {
```

Comment:
I don't see where we handle HibernernateException.

Reply:
add try-catch, to session-close